### PR TITLE
Add createsuperuser instruction

### PR DESCRIPTION
**Pull request by f-i-l-i-p**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,11 @@ python3 src/manage.py migrate
 python3 src/manage.py test api
 ```
 
+- Create a super user (Follow the instructions in the command. The email is not required):
```
Comment (Jack-Kolm) [inline README.md:55]:
```suggestion
- Create a super user, and make sure to follow the instructions in the command line (the email is not required):
```

File: README.md [hunk 2]
```diff
@@ -86,6 +91,11 @@ python src\manage.py migrate
 python src\manage.py test api
 ```
 
+- Create a super user (Follow the instructions in the command. The email is not required):
```
Comment (Jack-Kolm) [inline README.md:94]:
```suggestion
- Create a super user, and make sure to follow the instructions in the command line (the email is not required):
```

File: README.md [hunk 3]
```diff
@@ -86,6 +91,11 @@ python src\manage.py migrate
...
+```
+python3 src/manage.py createsuperuser
```
Comment (Jack-Kolm) [inline README.md:96]:
```suggestion
python src\manage.py createsuperuser
```

**Review by Jack-Kolm — COMMENTED** [review]
Consider my comments.

**Review by GoaGifflar — APPROVED** [review]

**Review by anton-bergman — APPROVED** [review]

**Pull request merged**